doc/spec/selection-button.md

```diff
@@ -0,0 +1,59 @@
+## Provides the suggested markup for the selection buttons
+
+* Specification for https://jira.ez.no/browse/EZP-27572
+
+ Goal here is to provide activable elements (like buttons) that are enabled and can triggers actions only if some elements of a list are selected.
+ The selection is done with the help of checkboxes or radio buttons related to items of the list.
+ Some buttons need at least one or more element checked to be enabled, but others will need one and only one element checked to be enabled.
+
+
+# Markup
+
```

Comment:
maybe a short sentence saying the behavior is implemented in a custom element `ez-selection-activate-element`